win32_event_log/README.md

```diff
@@ -252,7 +272,39 @@ The legacy method is the default mode for filters.
 <!-- xxz tab xxx -->
 <!-- xxx tab "Logs" xxx -->
 
-For each filter, add a log processing rule in the configuration file at `win32_event_log.d/conf.yaml`.
+You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. If you are using the Logs Tailer, the `query` option has better filtering performance than the `log_processing_rules` filters. The Datadog Agent cannot see any events filtered out by the `query` option. When using the `log_processing_rules` filters, the Agent can see every event and must process and format the event before applying the regex filter, so it takes more processing, and is less performant.
+
+You can use the `query` to filter events or filter out the bulk of the events with an [XPATH or structured XML query][21]. There is an expression limit on the syntax of XPath and XML queries. For additional filtering, use `log_processing_rules` filters.
+
+Datadog recommends creating and testing the query in Event Viewer's filter editor until the events shown in Event Viewer match what you want the Agent to collect.
+
+![Filter Current Log][23]
```

Comment:
I couldn't see this in your preview link, but what does the `!` do for this link

Reply:
This is the manual way to add images in READMEs that appear in GitHub and the Docs Site! We have to first add the images into the `/images` folder in this integration, and then it will appear in the preview link above 👍